CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.6.2] - 2017-04-19
```

Comment:
Yep, we definitely need to start rolling that out to the npm modules soon! Not today though :-)